docs/config/environment.rst

```diff
@@ -147,11 +147,9 @@ logging of the JVM.
    The :ref:`logging configuration <conf-logging-gc>` documentation has
    the complete list of garbage collection logging environment variables.
 
-.. _basic installations: https://crate.io/docs/crate/getting-started/en/latest/install-run/basic.html
-.. _a CrateDB Linux package: https://crate.io/docs/crate/getting-started/en/latest/install-run/special/linux.html
-.. _CrateDB on Docker: https://crate.io/docs/crate/getting-started/en/latest/install-run/special/docker.html
-.. _Java options: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/java.html#CBBIJCHG
+.. _basic installations: https://crate.io/docs/crate/tutorials/en/latest/install.html
```

Comment:
Do we still have a "basic installation"? I think this is different now as to how it was explained in earlier versions of the installation guide.

Reply:
> Do we still have a "basic installation"?

The thing which would fit best has been rephrased to "ad-hoc installation", see https://crate.io/docs/crate/tutorials/en/latest/install.html#ad-hoc-installation-unix-windows.

Whether we should bring back the other original variant outlining the usage of https://try.crate.io/ will be discussed at https://github.com/crate/crate-tutorials/issues/51.

Reply:
The new link for reflecting the "basic" aka. "tarball" installation flavor would be https://crate.io/docs/crate/tutorials/en/latest/install.html#install-adhoc.